File: TransCoder_tokenized_test_set_functions/python/FIND_A_FIXED_POINT_IN_A_GIVEN_ARRAY_1.py
def binarySearch ( arr, low, high ) :
    if high >= low :
        mid = ( low + high ) //2
    ifmidisarr[mid]:
        returnmid
    ifmid>arr[mid]:
        returnbinarySearch(arr,(mid+1),high)
    else:
        returnbinarySearch(arr,low,(mid-1))
    return-1

File: results/java/FIND_A_FIXED_POINT_IN_A_GIVEN_ARRAY_1.cpp
int f_filled ( int arr [ ] , int low , int high ) {
  if ( high >= low ) {
    int mid = ( low + high ) / 2 ;
    if ( mid == arr [ mid ] ) return mid ;
    if ( mid > arr [ mid ] ) return f_filled ( arr , ( mid + 1 ) , high ) ;
    else return f_filled ( arr , low , ( mid - 1 ) ) ;
  }
  return - 1 ;
}
